.github/workflows/benchmarks.yml

```
@@ -0,0 +1,48 @@
+name: Benchmarks
+
+on:
+  push:
+   branches:
+    - master
```

```suggestion
    - main
```

The default branch is different in this repo.